aea/configurations/schemas/definitions.json

```diff
@@ -87,7 +87,15 @@
     },
     "public_id": {
       "type": "string",
-      "pattern": "^[a-zA-Z_][a-zA-Z0-9_]{0,127}/[a-zA-Z_][a-zA-Z0-9_]{0,127}:(0|[1-9]\\d*)\\.(0|[1-9]\\d*)\\.(0|[1-9]\\d*)(?:-((?:0|[1-9]\\d*|\\d*[a-zA-Z-][0-9a-zA-Z-]*)(?:\\.(?:0|[1-9]\\d*|\\d*[a-zA-Z-][0-9a-zA-Z-]*))*))?(?:\\+([0-9a-zA-Z-]+(?:\\.[0-9a-zA-Z-]+)*))?$"
+      "pattern": "^[a-zA-Z_][a-zA-Z0-9_]{0,127}/[a-zA-Z_][a-zA-Z0-9_]{0,127}(:(0|[1-9]\\d*)\\.(0|[1-9]\\d*)\\.(0|[1-9]\\d*)(?:-((?:0|[1-9]\\d*|\\d*[a-zA-Z-][0-9a-zA-Z-]*)(?:\\.(?:0|[1-9]\\d*|\\d*[a-zA-Z-][0-9a-zA-Z-]*))*))?(?:\\+([0-9a-zA-Z-]+(?:\\.[0-9a-zA-Z-]+)*))?)?(:[a-zA-Z_][a-zA-Z0-9]{45})?$"
+    },
+    "ipfs_hash": {
+      "type": "string",
+      "pattern": "[a-zA-Z_][a-zA-Z0-9]{45}"
+    },
+    "extended_public_id": {
+      "type": "string",
+      "pattern": "^[a-zA-Z_][a-zA-Z0-9_]{0,127}/[a-zA-Z_][a-zA-Z0-9_]{0,127}(:(0|[1-9]\\d*)\\.(0|[1-9]\\d*)\\.(0|[1-9]\\d*)(?:-((?:0|[1-9]\\d*|\\d*[a-zA-Z-][0-9a-zA-Z-]*)(?:\\.(?:0|[1-9]\\d*|\\d*[a-zA-Z-][0-9a-zA-Z-]*))*))?(?:\\+([0-9a-zA-Z-]+(?:\\.[0-9a-zA-Z-]+)*))?)?(:[a-zA-Z_][a-zA-Z0-9]{45})?$"
     },
```

Comment:
Is it worth introducing the distinction here? We could also just extend the regex in the already backwards compatible way and then we don't need to update any of the paths in the other schemas.